packages/dds/tree/src/core/edit-manager/editManager.ts

```diff
@@ -309,6 +316,26 @@ export class EditManager<
 		return this.changeFamily.intoDelta(change);
 	}
 
+	/**
+	 * Fast-forwards the local branch to a new head commit that is ahead of the current local branch head.
+	 * @param newHead - the new head of the local branch. This must be a descendant of the current local branch head.
+	 * @returns the composition of changes from the previous head to the new head of the local branch
+	 */
+	public fastForwardLocalBranch(newHead: GraphCommit<TChangeset>): TChangeset {
+		const changes: GraphCommit<TChangeset>[] = [];
+		assert(
+			findAncestor([newHead, changes], (c) => c === this.localBranch) !== undefined,
```

Comment:
Shouldn't the ancestor always be `this.localBranch`?

Reply:
Yes, simplified.